docker/intel-dgpu-driver.Dockerfile

```diff
@@ -37,10 +38,17 @@ RUN git clone -b ${I915_RELEASE} --single-branch https://github.com/intel-gpu/in
     && cp defconfigs/drm .config \
     && make olddefconfig && make -j $(nproc) && make modules_install
 
+# Create /build/modules directory with all *.ko and *.ko.xz files
+RUN mkdir -p /build/modules && find /lib/modules/${KERNEL_FULL_VERSION}/ -name "*.ko" -type f | xargs -I {} cp {} /build/modules/ \
+    && find /lib/modules/${KERNEL_FULL_VERSION}/ -name "*.ko.xz" -type f | xargs -I {} cp {} /build/modules/ \
+    && install -D /lib/modules/${KERNEL_FULL_VERSION}/modules.order /build/modules/ \
+    && install -D /lib/modules/${KERNEL_FULL_VERSION}/modules.builtin /build/modules/
+
 # Firmware
 RUN git clone -b ${FIRMWARE_RELEASE} --single-branch https://github.com/intel-gpu/intel-gpu-firmware.git \
     && install -D /build/intel-gpu-firmware/COPYRIGHT /licenses/firmware/COPYRIGHT \
-    && install -D /build/intel-gpu-firmware/COPYRIGHT /build/intel-gpu-firmware/firmware/license/COPYRIGHT
+    && install -D /build/intel-gpu-firmware/COPYRIGHT /build/firmware/license/COPYRIGHT \
+    && cp -r /build/intel-gpu-firmware/firmware/dg2* /build/firmware/
```

Comment:
Can we use install command to replace cp command?

Reply:
We can try that. It should be the same from functionality perspective. @chaitanya1731, any thoughts?

Reply:
`install -D /build/intel-gpu-firmware/firmware/dg2* /build/firmware/` worked.